### PR TITLE
Upload desktop client releases to S3 only after successful merge to master

### PR DESCRIPTION
**Pull request by JonathanDCohen**
If necessary, a follow up to cache `awscli` can be written to save time.  We don't release often enough to justify it currently.

### CODE REVIEW

**Review by alalamav — APPROVED** [review]
👍

**Comment by fortuna** [comment]
Why can't I comment on this PR?

File: .github/workflows/upload_linux_client_to_s3.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Upload Linux Client to S3
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        run:
+          declare -a FILES=(
+            Outline-Client.AppImage
+            latest-linux.yml
+          )
+          for file in ${FILES[@]}; do
```
Comment (fortuna) [inline .github/workflows/upload_linux_client_to_s3.yml:48]: Proper escaping for files with spaces:
```suggestion
          for file in "${FILES[@]}"; do
```

File: .github/workflows/upload_windows_client_to_s3.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Upload Windows Client to S3
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        run:
+          declare -a FILES=(
+            Outline-Client.AppImage
+            latest.yml
+          )
+          for file in ${FILES[@]}; do
```
Comment (fortuna) [inline .github/workflows/upload_windows_client_to_s3.yml:48]:
```suggestion
          for file in "${FILES[@]}"; do
```

File: release_linux_client.sh [hunk 3]
```diff
@@ -18,18 +18,6 @@
...
-if [[ ! -f ~/.aws/credentials ]]; then
-  echo "No AWS credentials file found.  Follow the instructions in the README to create one"
-  exit 1
-fi
-if ! grep --quiet '\[outline-releases\]' ~/.aws/credentials ; then
-  echo "No outline-releases profile found in AWS credentials"
-  exit 1
-fi
-
 declare -a FILES=(
```
Comment (fortuna) [inline release_linux_client.sh:21]: Fix quoting below too.

File: release_windows_client.sh [hunk 4]
```diff
@@ -18,18 +18,6 @@
...
-if [[ ! -f ~/.aws/credentials ]]; then
-  echo "No AWS credentials file found.  Follow the instructions in the README to create one"
-  exit 1
-fi
-if ! grep --quiet '\[outline-releases\]' ~/.aws/credentials ; then
-  echo "No outline-releases profile found in AWS credentials"
-  exit 1
-fi
-
 declare -a FILES=(
```
Comment (fortuna) [inline release_windows_client.sh:21]: Fix quoting below.

**Review by fortuna — APPROVED** [review]
Thanks for the fix. Please fix the quoting to be safer.

**Pull request merged**